askama_shared/src/error.rs

```diff
@@ -51,12 +55,13 @@ impl std::error::Error for Error {
 
 impl Display for Error {
     fn fmt(&self, formatter: &mut fmt::Formatter<'_>) -> fmt::Result {
-        match *self {
-            Error::Fmt(ref err) => write!(formatter, "formatting error: {}", err),
+        match self {
+            Error::Fmt(err) => write!(formatter, "formatting error: {}", err),
+            Error::Custom(err) => write!(formatter, "formatting error: {}", err),
```

Comment:
This should probably have a different prefix?

Reply:
Do you think the prefix is needed at all? Probably I could just omit it. The contained err should have a useful message by itself.

Reply:
Seems fair.

Reply:
Removed.